.travis.yml

```diff
@@ -22,8 +20,7 @@ matrix:
   fast_finish: true
 
 before_script:
-  - rm -f ~/.phpenv/versions/$(phpenv version-name)/etc/conf.d/xdebug.ini
   - composer install --no-interaction --no-progress --prefer-dist
 
 script:
-  - vendor/bin/phpunit
+  - vendor/bin/phpunit -d error_reporting=16384
```

Comment:
Yes. Please refer this [issue](https://github.com/facebook/hhvm/issues/3033). 

Thanks.